src/Query.php

```diff
@@ -24,10 +24,6 @@ class Query extends Component implements QueryInterface
 
     public $type;
 
-    public $where;
```

Comment:
They are used in [QueryBuilder::build()](https://github.com/hiqdev/yii2-hiart/blob/master/src/QueryBuilder.php#L48-L51)